This file is part of MXE.
See index.html for further information.

workaround for -isystem flag messing up include order
https://bugzilla.redhat.com/show_bug.cgi?id=843436

diff -ur a/mingw-w64-headers/crt/float.h b/mingw-w64-headers/crt/float.h
--- a/mingw-w64-headers/crt/float.h	2014-01-11 21:38:50.640418500 +1100
+++ b/mingw-w64-headers/crt/float.h	2014-01-11 21:44:08.528393903 +1100
@@ -105,6 +105,15 @@
 	#define DBL_MAX_10_EXP	__DBL_MAX_10_EXP__
 	#define LDBL_MAX_10_EXP	__LDBL_MAX_10_EXP__
 
+    /* The difference between 1 and the least value greater than 1 that is
+    representable in the given floating point type, b**1-p.  */
+    #undef FLT_EPSILON
+    #undef DBL_EPSILON
+    #undef LDBL_EPSILON
+    #define FLT_EPSILON __FLT_EPSILON__
+    #define DBL_EPSILON __DBL_EPSILON__
+    #define LDBL_EPSILON    __LDBL_EPSILON__
+
 	/* Addition rounds to 0: zero, 1: nearest, 2: +inf, 3: -inf, -1: unknown.  */
 	/* ??? This is supposed to change with calls to fesetround in <fenv.h>.  */
 	#undef FLT_ROUNDS
